apps/calendar/js/controllers/notifications.js

```diff
@@ -55,26 +55,27 @@ function issueNotification(alarm) {
     }
 
     var begins = calc.dateFromTransport(busytime.start);
-    var distance = dateFormat.fromNow(begins);
-    var now = new Date();
+    return dateFormat.relativeDate(begins).then(distance => {
```

Comment:
@zbraniecki co has a very similar api to async/await. There are some exaples elsewhere in the calendar codebase too